rules/scripts/bitstreams_workspace.py

```diff
@@ -105,15 +105,18 @@ def NeedRefresh(self, interval):
         except FileNotFoundError:
             return True
 
-    def Get(self, file):
+    def Get(self, file, marker=''):
         """Perform an HTTP GET from the GCP bitstream bucket.
 
         Args:
             file: Filename in the bucket to retrieve.
+            marker: The last item returned in a previous query (i.e. where the
+                next query response should start). This parameter is used for
+                pagination.
         Returns:
             bytes
         """
-        response = urllib.request.urlopen(self.bucket_url + file)
+        response = urllib.request.urlopen(self.bucket_url + file + '?marker={}'.format(marker))
```

Comment:
We should probably be URL-encoding these parameters, but I suspect that's a no-op for the kinds of values we're feeding in. Meh, it's not too hard to call `urllib.parse.urlencode` to construct the query string. WDYT?

Reply:
Done!